language_detector/main.py

```diff
@@ -5,5 +5,21 @@
 
 def detect_language(text, languages=LANGUAGES):
     """Returns the detected language of given text."""
-    # implement your solution here
-    pass
+    counter = {}
+    for diction in languages:
+        counter[diction['name']] = 0
+        for word in diction['common_words']:
+            for text_word in text.split():
```

Comment:
Just a thought: I think it would be better to do the split outside of your loop just under where you initialise the counter variable. This way python would only need to do the work to split it up once. As you currently have it, python needs to split the paragraph up every time it gets to line 12.